src/App.tsx

```diff
@@ -47,18 +51,52 @@ const App = () => {
 
   // }
 
+  const initialFormFields = {
+    title: '',
+    img: '',
+    altText: '',
+    description: '',
+    url: ''
+  }
```

Comment:
I think we can move the setting the initial form fields into the Portfolio or add Portfolio entry components instead of making `App` larger than it already is.   App or the parent component of the form need to know the state, but they don't need to manually set the form fields to blank.